app-backend/backsplash/src/main/scala/io/Avro.scala

```diff
@@ -0,0 +1,187 @@
+package com.azavea.rf.backsplash.io
+
+import java.util.UUID
+
+import cats.data.{OptionT, NonEmptyList => NEL}
+import cats.effect.{IO, Timer}
+import cats.implicits._
+import com.azavea.rf.common.RollbarNotifier
+import com.azavea.rf.database.LayerAttributeDao
+import com.azavea.rf.datamodel.{MosaicDefinition, SingleBandOptions}
+import com.rf.azavea.backsplash.color
+import doobie.implicits._
+import geotrellis.raster.histogram._
+import geotrellis.raster.io.json.HistogramJsonFormats
+import geotrellis.raster.{Raster, io => _, _}
+import geotrellis.spark.io._
+import geotrellis.spark.io.postgres.PostgresAttributeStore
+import geotrellis.spark.io.s3.{S3CollectionLayerReader, S3ValueReader}
+import geotrellis.spark.tiling.LayoutLevel
+import geotrellis.spark.{SpatialKey, TileLayerMetadata, io => _, _}
+import geotrellis.vector.Extent
+import spray.json.DefaultJsonProtocol._
+import spray.json._
+
+import scala.util._
+
+object avro extends RollbarNotifier with HistogramJsonFormats {
```

Comment:
`Avro`?

Reply:
Not sure this meets the [condition](https://docs.scala-lang.org/style/naming-conventions.html) of

> An exception is when mimicking a package or function.